src/www/ui/api/Controllers/UploadController.php

```diff
@@ -376,7 +375,8 @@ public function deleteUpload($request, $response, $args)
    */
   public function moveUpload($request, $response, $args)
   {
-    $action = $request->getHeaderLine('action');
+    $queryParams = $request->getQueryParams();
+    $action = $queryParams['action'] ?? ""; 
```

Comment:
The API should still support V1 requests in deprecated mode until it is completely removed. You can read the version of request with `ApiVersion::getVersion($request) == ApiVersion::V2`.

See https://github.com/fossology/fossology/pull/2617/files#diff-1bae1ab555e908fff79a5b02ff0f05677a8d1f5653259e98531071acfeb7e16bR42 for reference.